draft-ietf-teep-architecture.md

```diff
@@ -639,9 +639,8 @@ the device can vary. The variations depend on whether the Untrusted Application
 together or are provided separately, and this has implications to the management of
 the TAs in a TEE. In addition to the Untrusted Application and TA(s), the TA(s) and/or TEE may also require additional data to personalize the TA to the device or a user.
 Implementations must support encryption of such
-Personalization Data to preserve the confidentiality of potentially
-sensitive data contained within it, and must support integrity protection
-of the Personalization Data.
+Personalization Data to preserve the confidentiality and integrity protection of potentially
```

Comment:
Delete "protection".
You preserve the integrity, you don't preserve protection per se.

Reply:
Fixed in a new PR 238.